test/integration/testdata/image-build/test-arg/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:latest
```

Comment:
`ubuntu:latest` is quite a large image, I'd recommend using a smaller image if you can, I'm sure there are a few options.

We use `gcr.io/k8s-minikube/busybox` for a few other tests, I'm not 100% sure if it has `echo` & `/bin/bash` though.